monitoring/stress/stress.js

```diff
@@ -0,0 +1,78 @@
+import http from 'k6/http';
+import { check, group, sleep, fail } from 'k6';
+
+// 서비스가 극한의 상황에서 어떻게 동작하는지 확인합니다.
+// 장기간 부하 발생에 대한 한계치를 확인하고 기능이 정상 동작하는지 확인합니다.
+// 최대 사용자 또는 최대 처리량을 확인합니다.
+// 스트레스 테스트 이후 시스템이 수동 개입없이 복구되는지 확인합니다.
+
+export let options = {
+	stages: [               
+    { duration: '1m', target: 0 },
+    { duration: '1m', target: 200 },
+    { duration: '1m', target: 400 },
+    { duration: '1m', target: 600 },
+    { duration: '1m', target: 800 },
+    { duration: '1m', target: 1000 },   
+    { duration: '1m', target: 0 },
```

Comment:
```suggestion
    { duration: '1m', target: 0 },
    { duration: '20s', target: 200 },
    { duration: '1m', target: 200 },    
    { duration: '20s', target: 400 },    
    { duration: '1m', target: 400 },
...
    { duration: '1m', target: 0 },
```

보통 부하테스트는 duration기간만큼 target을 증가시킵니다. 그래서 ramp-up 옵션이 있지 않으면 위와 같이 구성하셔야 해요